addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerJsonParser.java

```diff
@@ -347,6 +355,18 @@ private static ImageIcon createSvgIcon(URL url) throws Exception {
         return list;
     }
 
+    private boolean isValidQuery(String query) {
+        try {
+            QueryParser.parse(query);
+        } catch (SelectorParseException spe) {
+            patternErrorHandler.handleError(
+                    query, new PatternSyntaxException(spe.getMessage(), query, -1));
+            logger.info("Invalid Query Selector: {} : {}", query, spe.getMessage());
```

Comment:
The logging is probably no longer necessary, since any issues should be noticed prior to being released. Let me know if you agree.